lisa/tools/lsvmbus.py

```diff
@@ -0,0 +1,148 @@
+import re
+from typing import Any, List, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Debian, Linux, Redhat, Suse
+from lisa.util import LisaException
+
+from .wget import Wget
+
+PATTERN_VMBUSES = re.compile(r"(VMBUS ID[\w\W]*?)(?=VMBUS ID|\Z)", re.MULTILINE)
+PATTERN_VMBUS = re.compile(
+    r"^VMBUS ID\s+(?P<index>\d+): Class_ID = {?(?P<class_id>.+?)}?"
+    r" - \[?(?P<name>.+?)\]?\r$",
+    re.MULTILINE,
+)
+PATTERN_DEVICE = re.compile(
+    r"([\w\W]*?)Device_ID = {?(?P<device_id>.+?)}?\r$", re.MULTILINE
+)
+PATTERN_CHANNEL = re.compile(
```

Comment:
If the pattern is used in one class only, it's better to put it in that class. It will improve read ability.

Reply:
PATTERN_CHANNEL is used in two classes